sdk/mosaic.go

```diff
@@ -112,5 +139,32 @@ func (ref *MosaicService) GetMosaicNames(ctx context.Context, mosaicIds []*Mosai
 		return nil, err
 	}
 
-	return mosaicNameDTOsToMosaicNames(mscNamesDTO)
+	return dtos.toStruct()
+}
+
+func (ref *MosaicService) buildMosaicHierarchy(ctx context.Context, mscInfo *MosaicInfo) error {
+	if mscInfo == nil || mscInfo.Namespace == nil || namespaceIdToBigInt(mscInfo.Namespace.NamespaceId).Int64() == 0 {
```

Comment:
we must get a panic if namespaceIdToBigInt(mscInfo.Namespace.NamespaceId) is nil.

Reply:
Fixed